client/src/app/services/profile.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProfileService {
+
+  constructor() { }
+
+  publications: Array<any> = [];
+
+  getPublications() {
+    this.publications = [{
+      imageURL: "https://c.files.bbci.co.uk/48DD/production/_107435681_perro1.jpg",
```

Comment:
here you should grab the first image from the list of images in the Post interface